packages/bvaughn-architecture-demo/components/sources/PointPanel.tsx

```diff
@@ -49,14 +54,7 @@ export default function SourcePanelWrapper({
 
 function PointPanel({ className, point }: { className: string; point: Point }) {
   const { range: focusRange } = useContext(FocusContext);
-  const graphQLClient = useContext(GraphQLClientContext);
-  const { showCommentsPanel } = useContext(InspectorContext);
-  const { editPoint } = useContext(PointsContext);
   const client = useContext(ReplayClientContext);
-  const { accessToken, recordingId, trackEvent } = useContext(SessionContext);
-  const { executionPoint: currentExecutionPoint, time: curentTime } = useContext(TimelineContext);
-
-  const [showEditBreakpointNag, dismissEditBreakpointNag] = useNag(Nag.FIRST_BREAKPOINT_EDIT);
 
   const [hitPoints, hitPointStatus] = getHitPointsForLocationSuspense(
```

Comment:
Moved this call into its own component because I noticed that we were calling a method that potentially _suspended_ before calling hooks.